ntbs-service/wwwroot/source/Components/CascadingDropdowns.ts

```diff
@@ -12,15 +15,53 @@ const CascadingDropwdowns = Vue.extend({
             return response.data;
           })
           .then((data: any) => {
-            this.$refs["hospitalDropdown"].innerHTML = "<option value=''>Select Hospital</option>";
+            this.$refs["selectFieldHospitalId"].innerHTML = "<option value=''>Select Hospital</option>";
 
             data.forEach((item: any) => {
-              this.$refs["hospitalDropdown"].innerHTML += `<option value="${item.hospitalId}">${item.name}</option>`
+              this.$refs["selectFieldHospitalId"].innerHTML += `<option value="${item.hospitalId}">${item.name}</option>`
             });
           })
           .catch((error: any) => {
               console.log(error.response)
           });
+      },
+      validate: function(property : string) {
+          var shouldValidateFull = this.$props.shouldvalidatefull;
+          var modelName = this.$props.model;
+          // TODO: We need to check whether arrow ('=>') works in IE

```

Comment:
These arrow functions are fine - they get compiled. I verified that IE is not breaking on this page ;)